android/lib/src/main/java/com/swmansion/gesturehandler/ViewConfigurationHelper.kt

```diff
@@ -0,0 +1,10 @@
+package com.swmansion.gesturehandler
+
+import android.view.View
+import android.view.ViewGroup
+
+interface ViewConfigurationHelper {
+  fun getPointerEventsConfigForView(view: View): PointerEventsConfig?
+  fun getChildInDrawingOrderAtIndex(parent: ViewGroup, index: Int): View?
```

Comment:
In the implementation the return types are not nullable, shouldn't they also be there?
```suggestion
  fun getPointerEventsConfigForView(view: View): PointerEventsConfig
  fun getChildInDrawingOrderAtIndex(parent: ViewGroup, index: Int): View
```

Reply:
I checked usages and they're not expecting nulls, that's the right call.
Fixed in 75fba25.